src/Core/Manifest/ClassLoader.php

```diff
@@ -112,6 +112,13 @@ public function getItemPath($class)
                 break;
             }
         }
+
+        try {
```

Comment:
There's bits of logic that need to find the location of the test file ... for example to reference a fixture file relative to the test file.

However, if the file isn't track in the manifest, this return false. This extra bits attempts to use `ReflectionClass` to identify the location of the class.